query/cloudtrail/cloudtrail_enabled.sql

```diff
@@ -0,0 +1,28 @@
+with trails_enabled as (
```

Comment:
@sayan133 Is the intent of the query to check if all non-shadow trails that exist are enabled/logging? Or is it being used to check if the AWS account has at least one multi-region trail enabled?

Reply:
@cbruno10 In this query we are just checking whether the logging is enabled for the trails in our configured regions. For checking if the AWS account has at least one multi-region trail enabled, we have different control.